samples/bluetooth/mesh/dfu/distributor/README.rst

```diff
@@ -237,6 +237,26 @@ In this sample, the UART console is occupied by the shell module.
 Therefore, it uses SEGGER RTT as a logging backend.
 For the convenience, ``printk`` is also duplicated to SEGGER RTT.
 
+External flash support
+======================
+
+This sample supports external flash memory as secondary storage partition for saving of the firmware images, both as for self update as well as for distribution.
+See :ref:`ug_bootloader_external_flash` for more information about an external flash support as a partition in the :ref:`ug_bootloader_mcuboot_nsib`.
+Default configuration does not support external flash memory.
+To enable external flash support, set :makevar:`FILE_SUFFIX` to ``ext_flash`` when building the sample.
+
+Build the sample using the following command:
+
+.. code-block:: console
+
+   west build -p -b *board_name* -- -DFILE_SUFFIX=ext_flash
+
+.. note::
+   The external flash is not erased during the internal flash erasing procedure and requires separate options described in the ``nrfutil`` documentation.
+
+.. note::
+   The only external flash on ``nrf52840dk/nrf52840`` board is supported at the moment.
```

Comment:
Why can't we support nRF54L15DK which also has external flash? Shouldn't we target to new ICs?

Reply:
There are several reasons:
1. blob mesh io should be fixed first to use correct write-block-size
2. external flash and rram based flash emulation have different write-block-size
3. despite this is the same flash it is added on nrf54l on the different manner without qspi interface. That causes different KConfig options usage and potential incompatibility of image exchanging approach that is used with qspi. I did not investigate details yet.
For nrf54l we have separate task to add external flash to consider all differences correctly.